tox.setup.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34
+envlist = py27,py36
```

Comment:
Why aren't we supporting py34 and py35 any more? How hard would it be to support them?

Reply:
If we do, I'd like to add them as a separate test and also test 3.6. There was some issue (don't recall exactly) that worked in 3.4 but not in 3.6.

Reply:
I'd like to test on 2.7, 3.4, and 3.6 and/or 3.7. There are definitely people who will be pinned on 3.4 for a long time. <3.4 was kind of a mess if I recall correctly, and I think we're OK only supporting 2.7 and 3.4+.